Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -191,7 +203,38 @@ static bool HasPlugin(Assembly assembly)
             }
         }
 
-        private class PluginAssemblyLoadingContext : AssemblyLoadContext
+        private static bool IsExtensionAssembly(AssemblyMetadata assemblyMetadata)
+        {
+            return assemblyMetadata.AssemblyModelType.EqualsIgnoreCase(PluginConventions.Extension);
+        }
+
+        private static bool IsProfileAssembly(AssemblyMetadata assemblyMetadata)
+        {
+            return assemblyMetadata.AssemblyModelType.EqualsIgnoreCase(PluginConventions.Profile);
+        }
+
+        private static AssemblyMetadata LoadAssemblyMetadata(Assembly assembly)
+        {
+            var resourceName = assembly.GetManifestResourceNames()
+                .SingleOrDefault(p => p.EndsWith(AssemblyMetadataSearchString));
+            
+            if (resourceName == null)
+            {
+                throw new FileNotFoundException($"Assembly metadata file '{assembly.GetManifestResourceNames()}' not found for this project.");
+            }
+            var stream = assembly.GetManifestResourceStream(resourceName);
+
+            using (StreamReader reader = new StreamReader(stream))
+            {
+                _jsonFile = reader.ReadToEnd(); 
+            }
+
+            _logger.Debug($"Deserializing object type {typeof(AssemblyMetadata)} from file '{resourceName}'.");

```

Comment:
Missing single quotes around type parameter in message.

Reply:
You should also probably change the message to use "embedded resource" instead of "file" now.